### PR TITLE
[WIP] animate podring on sidebar for deployments

### PR DESCRIPTION
**Pull request by sahil143**
ODC-bug: https://jira.coreos.com/browse/ODC-2053

This PR adds animations for pod rings on the sidebar.
animation3 (3)

### CODE REVIEW

**Comment by christianvogt** [comment]
The ending transition isn't correct. The right ring is the one that should remain while the left one fades out and is removed as the right ring slides over to the left.

**Comment by andrewballantyne** [comment]
/hold cancel
/retitle [WIP] animate podring on sidebar for deployments

This PR is not finished. Putting the WIP label on it to distinguish that it is not ready for review.

@sahil143 please see Christian's comment.

File: frontend/packages/console-shared/src/components/pod/PodRingSet.tsx [hunk 1]
```diff
@@ -40,27 +42,43 @@ const PodRingSet: React.FC<PodRingSetProps> = ({ podData, resourceKind, obj, pat
...
+        )}
+      </TransitionGroup>
     </Split>
   );
 };
 
 export default PodRingSet;
+/**
+ *
+ */
```
Comment (TheRealJon) [inline frontend/packages/console-shared/src/components/pod/PodRingSet.tsx]: Remove empty multi-line comment

**Pull request closed without merging**